tools/checked-c-convert/ConstraintBuilder.cpp

```diff
@@ -57,51 +177,78 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
   // In any of these cases, due to conditional expressions, the number of
   // variables on the RHS could be 0 or more. We just do the same rule
   // for each pair of q_i to q_j \forall j in variables_on_rhs.
-  void constrainAssign(uint32_t V, Expr *RHS) {
+  //
+  // V is the set of constraint variables on the left hand side that we are
+  // assigning to.
+  void constrainAssign( std::set<ConstraintVariable*> V, 
```

Comment:
Could you amplify this comment some more? It's the set of constraint variables for the LHS. The LHS will have a pointer type and the constraint variables represent the constraints for that type.